base/missing.jl

```diff
@@ -107,16 +107,25 @@ max(::Missing, ::Any)     = missing
 max(::Any,     ::Missing) = missing
 
 # Rounding and related functions
-for f in (:(ceil), :(floor), :(round), :(trunc))
+round(::Missing, ::RoundingMode=RoundNearest; sigdigits::Integer=0, digits::Integer=0, base::Integer=0) = missing
+round(::Type{>:Missing}, ::Missing, ::RoundingMode=RoundNearest) = missing
+round(::Type{T}, ::Missing, ::RoundingMode=RoundNearest) where {T} =
+    throw(MissingException("cannot convert a missing value to type $T: use Union{$T, Missing} instead"))
+round(::Type{T}, x::Any, r::RoundingMode=RoundNearest) where {T>:Missing} = round(nonmissingtype(T), x, r)
+# to fix ambiguities
+round(::Type{T}, x::Rational, r::RoundingMode=RoundNearest) where {T>:Missing} = round(nonmissingtype(T), x, r)
+round(::Type{T}, x::Rational{Bool}, r::RoundingMode=RoundNearest) where {T>:Missing} = round(nonmissingtype(T), x, r)
```

Comment:
Is this one still needed?

Otherwise, looks good to me.

Reply:
They're still needed, according to ambiguity test failure below.

Reply:
Note I said "this one", i.e. the `Rational{Bool}` method which has no equivalent in the loop below.

Reply:
Yes. That one is still needed.

Reply:
I had a typo in my tests which led me to miss the ambiguity. I see it now. Would be nice if it could be removed without having to mention Rational in missing.jl, but I don't see a way.